### DISCUSSION TITLE
How to set a default route for `/admin`

### DISCUSSION THREAD

**Discussion by aaronzomback** [naymspace/backpex#723] [Q&A] [answered]
Hello,

First, thank you for making this great package - 

Perhaps I've missed this info in the guide - but how can we set a default route for the admin scope ?

When I access `/admin` it doesn't route by default to one of the live views - I get an error that the route does not exist. Of course it's only when I access one of the defined sub routes with corresponding live view, i.e. `/admin/posts/` that it works.

I'm still farely new to the Elixir/Phx world so bear with me if I am missing something obvious.

Thanks for any insight.

Aaron

**Comment by Flo0807** [comment, marked as answer]
Hey @aaronzomback!

You can create a (redirect) controller for this use case.

In your `router.ex` file:

```elixir
scope "/admin", MyAppWeb do
  get "/", RedirectController, :redirect_to_posts
end
```

For example, in `my_app_web/controller` create a file named `redirect_controller.ex`:

```elixir
defmodule MyAppWeb.RedirectController do
  use MyAppWeb, :controller

  def redirect_to_posts(conn, _params) do
    conn
    |> Phoenix.Controller.redirect(to: ~p"/admin/posts")
    |> Plug.Conn.halt()
  end
end
```

That's it!

**Reply by aaronzomback** [reply]
Hello @Flo0807  thanks so much, that solved it! I totally missed this line here: 
https://github.com/naymspace/backpex/blob/b9a1654791d9304d7f49fb4c5b9f1963c1bb4f3a/demo/lib/demo_web/router.ex#L20

I would be happy to contribute perhaps a minor update to the documentation to point this out to new users - for me it wasn't completely obvious. Let me know if that's something that might interest you.

Cheers!

**Reply by Flo0807** [reply]
> Let me know if that's something that might interest you.

Yes! Let's add a note to the docs so that others don't run into this problem. Feel free to open a Pull Request.

**Comment by aaronzomback** [comment]
@Flo0807 here it is https://github.com/naymspace/backpex/pull/724 thanks again
